README.md

```diff
@@ -10,28 +10,26 @@ These Intel Data Center hardware features currently include:
 
 The following features will be included in future releases.  
 - Intel® Data Streaming Accelerator (Intel® DSA) 
-- Intel® In-Memory Analytics Accelerator (Intel® IAA) 
-- Intel® FPGA N6000 
 
-See details about [Supported Intel Hardware features](/docs/supported_platforms.md#supported-intel-hardware-features) and [Supported RHOCP Versions](/docs/supported_platforms.md#supported-rhocp-versions).  
+Intel AI hardware and optimized software solutions are integrated into Red Hat OpenShift AI for ease of provisioning and configuration. The Habana AI Operator is used to provision Intel® Gaudi® accelerators and released on the Red Hat Ecosystem Catalog.
```

Comment:
Habana AI operator should be a link to https://catalog.redhat.com/software/container-stacks/detail/64342b3bcbfbb9a6588ce8dd?gs&q=habana